l10n_nl_partner_salutation/models/res_partner.py

```diff
@@ -3,7 +3,21 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 from odoo import api, models, fields
 
-GENDERS = [('male', 'Male'), ('female', 'Female'), ('unknown', 'Unknown')]
+
+DEFAULT_PREFIX = {
+    'shortcut': {
+        'male': 'De heer',
```

Comment:
what i disliked with the 8 version is that we'll never be able to address non dutch speaking people with the dutch regime. Better put this in a function of its own to return the dictionary, then you can wrap the values in `_()`. And you also enable other modules to hook in there and do what they think is the good default.